doc/navigating.rst

```diff
@@ -290,13 +128,13 @@ can use the *Ignored Files Editor*.
 
```

Comment:
Maybe "The *Ignored Files Editor* can be opened by clicking on the button in the General tab of the settings dialog. The settings apply to all configured accounts.